### PR TITLE
[DS-3804] Jar dependency issues running DSpace 7x REST

### PR DESCRIPTION
**Pull request by terrywbrady**
### This PR is a temporary fix to restore stability to the REST build process.
- https://jira.duraspace.org/browse/DS-3804
- https://jira.duraspace.org/browse/DS-3805

### Recommended approach
The more comprehensive approach to the problem will be to upgrade the version of log4j that is in use.  DSpace currently uses a version that is at end of life.
- https://jira.duraspace.org/browse/DS-3135
- https://docs.spring.io/spring-boot/docs/current/reference/html/howto-logging.html#howto-configure-log4j-for-logging

### CODE REVIEW

File: dspace-spring-rest/pom.xml [hunk 1]
```diff
@@ -169,6 +169,20 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 			<version>${spring-boot.version}</version>
+                        <exclusions>
```
Comment (tdonohue) [inline dspace-spring-rest/pom.xml]: Could we add a simple comment here that says something like:

**Review by tdonohue — APPROVED** [review]
Looks good overall. One minor request to add in a comment about the temporary dependency exclusions. But, after that, I think this is good to merge

File: dspace-oai/pom.xml [hunk 2]
```diff
@@ -159,6 +159,10 @@
                      <groupId>org.apache.commons</groupId>
                      <artifactId>commons-lang3</artifactId>
                 </exclusion>
+                <exclusion>
```
Comment (abollini) [inline dspace-oai/pom.xml]: I'm a bit worried about this exclusion as it is an artifact from an old, different layout, jackson version. The mapper class are now in the jackson-databinder artifact so if we really use this staff we could hit a classnotfound exception at runtime somewhere... BTW the jtwig-spring version in our depency is quite old the new one is: http://jtwig.org/

**Review by abollini — COMMENTED** [review]
if the jackson exclusion is not required to have the OAI running properly I will prefer to skip it

**Pull request merged**